.github/workflows/deploy.yml

```diff
@@ -46,14 +47,31 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Set environment-specific variables
+        id: set_vars
+        run: |
+          if [ "${{ inputs.github_environment }}" = "prod" ]; then
+            echo "KUBE_CLUSTER=${{ secrets.PRODUCTION_KUBE_CLUSTER }}" >> $GITHUB_ENV
+            echo "KUBE_NAMESPACE=${{ secrets.PRODUCTION_KUBE_NAMESPACE }}" >> $GITHUB_ENV
+            echo "KUBE_TOKEN=${{ secrets.PRODUCTION_KUBE_TOKEN }}" >> $GITHUB_ENV
+            echo "KUBE_CERT=${{ secrets.PRODUCTION_KUBE_CERT }}" >> $GITHUB_ENV
+            echo "KUBE_API=https://${{ secrets.PRODUCTION_KUBE_CLUSTER }}" >> $GITHUB_ENV
+          else
+            echo "KUBE_CLUSTER=${{ secrets.DEVELOPMENT_KUBE_CLUSTER }}" >> $GITHUB_ENV
+            echo "KUBE_NAMESPACE=${{ secrets.DEVELOPMENT_KUBE_NAMESPACE }}" >> $GITHUB_ENV
+            echo "KUBE_TOKEN=${{ secrets.DEVELOPMENT_KUBE_TOKEN }}" >> $GITHUB_ENV
+            echo "KUBE_CERT=${{ secrets.DEVELOPMENT_KUBE_CERT }}" >> $GITHUB_ENV
+            echo "KUBE_API=https://${{ secrets.DEVELOPMENT_KUBE_CLUSTER }}" >> $GITHUB_ENV
+          fi 
+            
```

Comment:
None of this section is needed, if you update the cloudplatform terraform code here, github.com/ministryofjustice/cloud-platform-terraform-serviceaccount?ref=1.0.0  - remove the environment string interpolation.

Reply:
updated, (none of these were part of the original PR, I had some commits and lost some during squash)